### DISCUSSION TITLE
Trigger JumpToPage on pressing enter

### DISCUSSION THREAD

**Discussion by tom-ramesh** [primefaces/community#2501] [PrimeReact] [answered]
Hello, 
I'm working on getting our app to have a jump to page feature. The current issue is that it goes to page 1 and then page 12 when we try to move to a page with two digits (for example, 12). Thus, two API calls are being made. I've seen an example of how to accomplish this with CurrentPageReport. However, because JumpToPageIndex makes the most sense, can we accomplish this with it? 

[StackBlitz](https://stackblitz.com/edit/j1ammf?file=src%2FApp.jsx)

**Comment by melloware** [comment, marked as answer]
Issue open: https://github.com/primefaces/primereact/issues/7060
